solutions/observability/apps/monitor-resources-on-private-networks.md

```diff
@@ -103,10 +103,13 @@ When the {{agent}} is running you can add a new {{private-location}} in the UI:
 
 1. Find `Synthetics` in the [global search field](/explore-analyze/find-and-organize/find-apps-and-objects.md).
 1. Go to **Settings**.
-2. Go to the **{{private-location}}s** tab.
-3. Click **Add location**.
-4. Give your new location a unique *Location name* and select the *Agent policy* you created above.
-5. Click **Save**.
+1. Go to the **{{private-location}}s** tab.
+1. Click **Add location**.
```

Comment:
```suggestion
1. Click **Create location**.
```
Looks like it's **Create location** now.

![image](https://github.com/user-attachments/assets/6ca070d9-2456-4cb2-8e56-ed4b9dbea871)